primefaces/src/main/java/org/primefaces/component/chips/ChipsRenderer.java

```diff
@@ -64,6 +64,9 @@ public void decode(FacesContext context, UIComponent component) {
         }
 
         if (submittedValues.length > 0) {
+            if (chips.isUnique()) {
+                submittedValues = Arrays.stream(submittedValues).distinct().toArray(String[]::new);
```

Comment:
No big deal, just a bit shorter:
```suggestion
                submittedValues = Stream.of(submittedValues).distinct().toArray(String[]::new);
```